app/requests/remote/push_notification.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import moment from 'moment-timezone';
+import {IntlShape} from 'react-intl';
+
+import {Client4} from '@client/rest';
+import PushNotifications from '@init/push_notifications';
+import {getCommonSystemValues} from '@queries/system';
+import {getSessions} from '@requests/remote/user';
+import {Config} from '@typings/database/config';
+import {getActiveServerDatabase} from '@utils/database';
+import {isMinimumServerVersion} from '@utils/helpers';
+
+const MAJOR_VERSION = 5;
+const MINOR_VERSION = 24;
```

Comment:
we don't need to check for this anymore as the latest ESR is now above this version, 5.24 is no longer supported, let's consider adding this in the `sessions` remote requests or something